frontend/views/containers/sidebar/GroupsMinIncome.vue

```diff
@@ -10,27 +10,52 @@ div
     :aria-label='L("Change Mincome")'
     @click='openProposal'
   ) Change
+  br
+  i18n.link(
+    tag='button'
+    @click='changeMin'
+  ) Increase {{ currency }}10
```

Comment:
i18n tags do not support Vuejs templates like this. Use { instead of {{ and see examples for how to do it properly